Gemfile

```diff
@@ -5,6 +5,7 @@ git_source(:github) do |repo_name|
   "https://github.com/#{repo_name}.git"
 end
 
+gem 'figaro'
```

Comment:
@erose357 @swdonovan @jtruong2 @markyv18 `EVERYBODY BUNDLE AND RUN 'figaro install' WHEN WE PULL DOWN FROM MASTER` 😄